README.md

```diff
@@ -1158,9 +1158,9 @@ Adjust.AppDidLaunch(config);
 **For Android app:**
 
 ```cs
-AdjustConfig adjustConfig = AdjustConfig(this, appToken, environment);
-adjustConfig.SetAppSecret(secretId, info1, info2, info3, info4);
-Adjust.OnCreate(adjustConfig);
+AdjustConfig config = AdjustConfig(this, appToken, environment);
+config.SetAppSecret(secretId, info1, info2, info3, info4);
+Adjust.OnCreate(config);
```

Comment:
Not consistent with the rest of android codes
adjustConfig → config

Reply:
I agree. Snippet should still be functional, but nice to have iOS and Android ones synced in naming. 👌